extensions/sql-database-projects/src/test/baselines/openSdkStyleSqlProjectWithGlobsSpecifiedBaseline.xml

```diff
@@ -28,7 +28,7 @@
     <None Include="..\other\folder2\Script.PostDeployment2.sql" />
   </ItemGroup>
   <ItemGroup>
-    <ArtifactReference Condition="'$(NetCoreBuild)' == 'true'" Include="$(NETCoreTargetsPath)\SystemDacpacs\150\master.dacpac">
+    <ArtifactReference Condition="'$(NetCoreBuild)' == 'true'" Include="$(SystemDacpacsLocation)\SystemDacpacs\150\master.dacpac">
```

Comment:
not related to the PR - we probably need a code path when target platform changes, we fix the version 150/160 etc in reference path. (or we need a test of that code path is already there)